EIC-EA/EIC-community-D8
# Feature/eicnet 765 hide group features on creation form

Testing procedure:

- Create a group
- The "Discussions" group menu item should be displayed
- Edit the group => Discussions feature should be enabled
- Edit group permissions => "Access Discussions overview" permission should be enable for Member role

- [ ] Need to check why config doens't have uuid
